Resources/public/js/alloyeditor/toolbars/config/block-base.js

```diff
@@ -16,10 +16,12 @@ YUI.add('ez-alloyeditor-toolbar-config-block-base', function (Y) {
     var ReactDOM = Y.eZ.AlloyEditor.ReactDOM;
 
     function outlineTotalWidth(block) {
-        return (
-            parseInt(block.getComputedStyle('outline-offset'), 10) +
-            parseInt(block.getComputedStyle('outline-width'), 10)
-        );
+        var outlineOffset = parseInt(block.getComputedStyle('outline-offset'), 10);
+        var outlineWidth = parseInt(block.getComputedStyle('outline-width'), 10);
+        if ( isNaN(outlineOffset) ) {
+            outlineOffset = 1;
```

Comment:
1 because that was the value I got in chrome and firefox everytime I inspected the value